content/documentation/staging/quick-start/index.adoc

```diff
@@ -16,108 +16,59 @@ weight: 2
 :imagesdir: /images/quickinstall/
 :sectlinks:
 
-icon:lightbulb[size=1x]{nbsp} This page is intended for users installing Kiali for the first time, not recommended for custom production environments. In case of custom installations, visit the link:{{< ref "/documentation/staging/installation-guide" >}}[installation guide].
+icon:lightbulb[size=1x]{nbsp} This page is intended for users installing Kiali for the first time. These instructions are not recommended to install Kiali in production environments. In the case of custom installations, visit the link:{{< ref "/documentation/staging/installation-guide" >}}[installation guide].
 
-== Installing Kiali
+icon:lightbulb[size=1x]{nbsp} If you are installing on OpenShift, you must grant the `cluster-admin` role to the user that is installing Istio and Kiali. If OpenShift is installed locally on the machine you are using, the following command should log you in as user `system:admin` which has this `cluster-admin` role: `oc login -u system:admin`
 
-=== Install via OperatorHub
+icon:lightbulb[size=1x]{nbsp} Before attempting to install Kiali, please review the link:../installation-guide/#_prerequisites[prerequisites].
 
-This is the simplest install mechanism. If you are running OpenShift (or you have a Kubernetes environment with OperatorHub installed), simply go to the OperatorHub console and choose to install the Kiali Operator.
+== Install via Kiali Server Helm Chart
```

Comment:
I suggest to use a previous point like 

```
== Install via Istio addons

From Istio 1.7 Kiali addon can be installed from:
$ kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.7/samples/addons/kiali.yaml
```

Then, if I want to install via Helm is for two potential reasons:

a) I didn't find how to do that from the Istio page
b) I want to uninstall and install a new Kiali version different from the default one (and then I want to continue reading the helm install point).

Reply:
I do not like putting in version-specific instructions here. Once 1.8+ is released, these docs immediately become outdated and we'll have to remember to change these. 

Regarding this point:

> a) I didn't find how to do that from the Istio page

Instead of putting those instructions here, we should just have a link to that Istio page to help the person find it. That way we address two things: (1) we can help solve the "I didn't find it from the Istio page" problem and (2) we therefore do not have a copy or a repeat of the Istio docs.

Reply:
To me, Istio.io and Kiali.io share this and it should be there.
Otherwise we force the new comers to understand the integration relation from the product and I don't think we want that.

To update this for 1.7 and 1.8 seems a very low effort but to have a one line to install things in the quickest way.

Reply:
we can add that - but realize there is no migration between that and our helm. So they will have to know to uninstall first before upgrading kiali via helm. I do have that warning, but of course i'm sure some people will not do it :)

Reply:
OK. I added something that I think addresses all of our concerns.